apps/kitchen-sink/src/ensemble/screens/home.yaml

```diff
@@ -1,8 +1,4 @@
 View:
-  onLoad:
-    executeCode: |
-      ensemble.storage.set('inputVal', 'sagar')
-
```

Comment:
Can we move this example to the Actions page and show usage of calling multiple APIs and a time?